pull-requests.md

```diff
@@ -22,6 +16,22 @@ The content changes go through the following stages before they are published on
 2. **Review:** Your changes are reviewed by [MDN members and volunteers](#pull-request-review-process).
 3. **Publishing:** Updated content goes live within a day of merging via a site rebuild once every 24 hours.
 
+## Before you start
+
+Make sure that you're not making changes that someone else is already working on.
+You can find tasks under 'Issues' in any of the [MDN repositories](https://github.com/orgs/mdn/repositories) (for example, [`mdn/content` issues](https://github.com/mdn/content/issues)) and our [public GitHub project boards](https://github.com/orgs/mdn/projects).
```

Comment:
I feel there are two scenarios here that we should differentiate. The first is a scenario in which contributors already know what they want to update. In this case, they should cross-check if there is already an open issue or PR. The second is a scenario when they are wanting to make contributions. In that case, they can go through open issues and possibly filter by 'good first issue' label. 

```suggestion
In the [MDN repository](https://github.com/orgs/mdn/repositories) you want to work, check out the 'Issues' and 'Pull requests' tabs to confirm that an issue or pull request does not already exist related to the work you want to do.

If you are looking to contribute to the project, you can find tasks under 'Issues' in any of the [ (for example, [`mdn/content` issues](https://github.com/mdn/content/issues)) and our [public GitHub project boards](https://github.com/orgs/mdn/projects).
```